ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/rewriter/RemoveAndBypassNode.scala

```diff
@@ -0,0 +1,68 @@
+package pl.touk.nussknacker.restmodel.displayedgraph.rewriter
+
+import pl.touk.nussknacker.engine.graph.node.{NodeData, Sink}
+import pl.touk.nussknacker.restmodel.displayedgraph.{DisplayableProcess, ProcessRewriter}
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.Edge
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.EdgeType.FilterFalse
+
+
+object RemoveAndBypassNode {
+  private case class NodeEdges(incoming: List[Edge], outgoing: List[Edge])
+}
+
+class RemoveAndBypassNode(process: DisplayableProcess, node: NodeData) {
+  import RemoveAndBypassNode._
+
+  def apply: DisplayableProcess = {
+    val nodeEdges = getNodeEdges(node)
+
+    getBranchRootToRemove(nodeEdges) match {
+      case Some(branchRoot) =>
+        val processWithRemovedBranches = ProcessRewriter.removeBranch(process, branchRoot)
+        new RemoveAndBypassNode(processWithRemovedBranches, node).apply
+
+      case None =>
+        val maybeBypassEdge = getBypassEdge(nodeEdges)
+
+        val newEdges = maybeBypassEdge.map(List(_)).getOrElse(List.empty) ++
+          process.edges.filterNot { disabledEdge =>
+            (nodeEdges.incoming ++ nodeEdges.outgoing).contains(disabledEdge)
+          }
+
+        process.copy(
+          nodes = process.nodes.filterNot(_ == node),
+          edges = newEdges)
+    }
+  }
+
+  private def getBranchRootToRemove(nodeEdges: NodeEdges): Option[NodeData] =
```

Comment:
What do you think about skipping "get" prefixes in methods in this class? We rather sticking with scala-way approach - without those prefixes... Or replace with `find` prefix.